src/test/java/com/rultor/agents/daemons/StartsDaemonITCase.java

```diff
@@ -81,6 +85,70 @@
     @Test
     public void startsDaemon() throws Exception {
         Assume.assumeFalse(SystemUtils.IS_OS_WINDOWS);
+        final Talk talk = this.talk();
+        MatcherAssert.assertThat(
+            talk.read(),
+            XhtmlMatchers.hasXPaths(
+                "/talk/daemon[started and dir]",
+                "/talk/daemon[ends-with(started,'Z')]"
+            )
+        );
+        final String dir = talk.read().xpath("/talk/daemon/dir/text()").get(0);
+        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        TimeUnit.SECONDS.sleep(2L);
```

Comment:
@sebing please read http://www.kyleblaney.com/junit-best-practices/ - ideally we shouldn't have any sleeps in unit tests